### PR TITLE
Cicd running tests on all os

### PR DESCRIPTION
**Pull request by Avvessalom**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+jobs:
+
+  build-launcher-linux:
+
+    runs-on: ubuntu-latest
```
Comment (SerVB) [inline .github/workflows/build.yml:13]: Why don't we build on other platforms?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -11,13 +22,3 @@ jobs:
           java-version: '11'
       - run: ./gradlew clean
       - run: ./gradlew check build
```
Comment (SerVB) [inline .github/workflows/test.yml:24]: I guess we should make configurations consistent between repos. [Here](https://github.com/JetBrains/projector-server/pull/34/files) we've split check and build to two different workflows.

**Review by SerVB — CHANGES_REQUESTED** [review]
Let's also add some badges to README.

File: .github/workflows/build.yml (hunk 1 above)
Comment (Avvessalom) [reply .github/workflows/build.yml:13]: they were commented out, but I removed them later. For Windows and Mac, build does not work now

File: .github/workflows/test.yml (hunk 2 above)
Comment (Avvessalom) [reply .github/workflows/test.yml:24]: yes, it will probably be correct. earlier in this repository, I split, but after that I even returned it as it was

**Comment by Avvessalom** [comment]
if after the corrections the PR suits you, please do not confirm, I need to correct the commit history

**Comment by Avvessalom** [comment]
@SerVB please fix the merge conflict, I do not have enough access

**Review by SerVB — CHANGES_REQUESTED** [review]
Well, you should fix the conflict locally (probably via rebasing to the origin/master and force pushing here). Feel free to DM me if you need help with this.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+#
+#      steps:
+#        - uses: actions/checkout@v2
+#        - uses: actions/setup-java@v1
+#          with:
+#            java-version: '11'
+#        - name: check projector-launcher building windows
+#          run: |
+#            npm cache clean --force
+#            ./gradlew :projector-launcher:distZip
```
Comment (SerVB) [inline .github/workflows/build.yml:64]: Can we at least build OS-specific variants like `:packageZipDarwinX64` and `:packageZipWin32X64` on these OSes?

**Review by SerVB — CHANGES_REQUESTED** [review]

**Review by SerVB — APPROVED** [review]

**Pull request merged**